site-docker.yml.sample

```diff
@@ -15,12 +15,16 @@
   - mgrs
   gather_facts: false
   become: True
+  vars:
+    delegate_facts_host: True
   pre_tasks:
     - name: gather and delegate facts
       setup:
       delegate_to: "{{ item }}"
       delegate_facts: True
       with_items: "{{ groups['all'] }}"
+      tags: always
+      when: delegate_facts_host | bool
```

Comment:
add the opposite too when delegate_facts_host is false we still need to gather facts